src/main/java/nerdhub/textilelib/event/client/render/hud/DrawHudCallback.java

```diff
@@ -0,0 +1,23 @@
+package nerdhub.textilelib.event.client.render.hud;
+
+import net.minecraft.client.gui.DrawableHelper;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+@Environment(EnvType.CLIENT)
+public interface DrawHudCallback {
+
+    Event<DrawHudCallback> EVENT = EventFactory.createArrayBacked(DrawHudCallback.class, listeners -> (type, hud, mouseX, mouseY, deltaTime) -> {
+        for (DrawHudCallback callback : listeners) {
+            callback.drawHud(type, hud, mouseX, mouseY, deltaTime);
+        }
```

Comment:
should also be cancelable

Reply:
Currently these events are called AFTER the main draw for the hud, maybe we should have a BEFORE and AFTER event?

Reply:
yes